recipes/newrelic/apm/php-agent/php.yml

```diff
@@ -67,3 +70,47 @@ install:
           else
             echo "You will need to restart your PHP application in order for instrumentation to be enabled."
           fi
+
+    send_transaction:
+      cmds:
+        - echo "Send queryable transactions."
+        - |
+          TMP_INSTALL_DIR="/tmp/php-agent-install"
+          cd $TMP_INSTALL_DIR
+          #
+          # Expand the log files from the most recent installation attempt,
+          # and get the name of the most recent log file.
+          #
+          tar -xvf $(ls -t nrinstall*.tar | head -1)
+          NR_INSTALL_LOG=$(ls -t nrinstall*.log | head -1)
```

Comment:
We should guard against a missing tar file or log file. Unlikely to happen, but may result in us following the non-web path when there is actually an installation problem.